pom.xml

```diff
@@ -32,8 +32,8 @@
         <!-- Versioning -->
         <version.arquillian_core>1.1.11.Final</version.arquillian_core>
         
-        <weld.api.version>2.3.SP2</weld.api.version>
-        <weld.core.version>2.3.5.Final</weld.core.version>
+        <weld.api.version>2.4.CR1</weld.api.version>
+        <weld.core.version>2.4.0.CR1</weld.core.version>
```

Comment:
Sorry I don't follow. You can either use current `Beta2` version (with 2.3.5.Final) or wait for next release with newer Weld version (I think I can wait for 2.4.0.Final).